pageserver/src/config.rs

```diff
@@ -194,6 +194,13 @@ pub struct PageServerConf {
     /// Interpreted protocol feature: if enabled, validate that the logical WAL received from
     /// safekeepers does not have gaps.
     pub validate_wal_contiguity: bool,
+
+    /// When set, the previously written to disk heatmap is loaded on tenant attach and used
+    /// to avoid clobbering the heatmap from new, cold, attached locations.
```

Comment:
we still do a merging here, right?

Reply:
Merging of heatmaps you mean? When the flag is enabled the previous heatmap is taken into consideration, yes.